src/Addon/Addonable.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Give\Addon;
+
+interface Addonable {
```

Comment:
Please document what this interface is for and when the appropriate time is to use it.

Reply:
This has been removed